src/core/pageContent/utils.ts

```diff
@@ -123,6 +123,9 @@ export function getEventCardProps(
     clampText: true,
     direction: 'responsive' as CardProps['direction'],
     openLinkInNewTab: false,
+    withTitleIcon: ['ahjo', 'helsinki'].includes(
```

Comment:
Maybe `['ahjo', 'helsinki']` could be moved to a named constant like in [events-helsinki-monorepo](https://github.com/City-of-Helsinki/events-helsinki-monorepo/blob/sports-helsinki-v1.13.0/packages/components/src/constants/event-constants.ts#L78-L88):
```typescript
/**
 * Prefixes of organization IDs under City of Helsinki root organization 'ahjo:00001',
 * @see https://api.hel.fi/linkedevents/v1/organization/ahjo:00001/
 *
 * Kultus subtree of the organization tree starts at 'kultus:0' and is excluded,
 * @see https://api.hel.fi/linkedevents/v1/organization/kultus:0/
 */
export const CITY_OF_HELSINKI_LINKED_EVENTS_ORGANIZATION_PREFIXES = [
  'ahjo',
  'helsinki',
];
```
Or it could perhaps even be given as a parameter like `CheckMarkIcon` so the logic would be more robust to change?

Reply:
i was thinking about it. i can fix

Reply:
fixed